assets/js/base/components/cart-checkout/address-form/address-form.tsx

```diff
@@ -42,7 +42,7 @@ const validateShippingCountry = (
 	clearValidationError: ( error: string ) => void,
 	hasValidationError: boolean
 ): void => {
-	const validationErrorId = 'shipping-missing-country';
+	const validationErrorId = 'shipping_country';
```

Comment:
Basing the error ID on the field name allows us to look it up later in the validation store.